spiped/main.c

```diff
@@ -198,6 +205,19 @@ main(int argc, char * argv[])
 		}
 	}
 
+	/* Check whether we are init (i.e. running inside a container). */
```

Comment:
We aren't init.  But we might be *running as* init.  I'd suggest:

    /* Check whether we are running as init (e.g., inside a container). */